### ISSUE TITLE
ArgumentException when using access token with prefix

### ISSUE THREAD

**Issue by ap0llo** [nmklotas/GitLabApiClient#204]
GitLab allows setting a prefix for access tokens (see https://docs.gitlab.com/ee/user/admin_area/settings/account_and_limit_settings.html#personal-access-token-prefix).
However, it seems access tokens with a prefix cannot be used in GitLabApiClient.

## Steps to reproduce

- Create a Access Token in a GitLab instance that has a prefix configured, the access token might look something like this: `prefix-abcde12345efghi67890`
- Initialize a new `GitLabClient` with the access token:
  ```cs
  var client = new GitLabClient("https://example.com", "prefix-abcde12345efghi67890");	
  ```

### Expected Behavior

A new `GitLabClient` is initialized

### Actual Behavior

An ArgumentException is thrown:

```txt
System.ArgumentException: Unsupported authentication token provide, please private an oauth or private token
   at GitLabApiClient.Internal.Http.GitLabHttpFacade..ctor(String hostUrl, RequestsJsonSerializer jsonSerializer, String authenticationToken, HttpMessageHandler httpMessageHandler, Nullable`1 clientTimeout)
   at GitLabApiClient.GitLabClient..ctor(String hostUrl, String authenticationToken, HttpMessageHandler httpMessageHandler, Nullable`1 clientTimeout)
```

---

I looked into the implementation and the exception is thrown by [this check in GitLabHttpFacade](https://github.com/nmklotas/GitLabApiClient/blob/cc79852aba4e1cb06154d0da7fcd19259b49181e/src/GitLabApiClient/Internal/Http/GitLabHttpFacade.cs#L35) that expects access tokens to be exactly 20 characters long.

I'd be happy to provide a PR to fix this - I am however not sure how to decide whether to use the `PRIVATE-TOKEN` header or the Bearer Authorization header.
The easiest fix would be to treat any non-empty string, that is not 64 characters long, as access token, but I'm not sure if this would break something else.

**Comment by fgreinacher** [comment]
@ap0llo This should be solved now that https://github.com/nmklotas/GitLabApiClient/pull/207 has been merged 🥳

**Comment by jetersen** [comment]
released in 1.8.0
